docs/design/design-doc-process.md

```diff
@@ -0,0 +1,27 @@
+Design doc process:
```

Comment:
Why are there changes to this?

Reply:
This change adds a completely new file. The file was already fixed and moved in another PR.